### PR TITLE
OPSEXP-2929 Point links to support.hyland.com

### PR DESCRIPTION
**Pull request by pmacius**
Ref: OPSEXP-2929

### CODE REVIEW

**Comment by pmacius** [comment]
Added a jira comment with the according pages that I couldn't find on new website: https://hyland.atlassian.net/browse/OPSEXP-2929?focusedCommentId=2228348

File: docs/helm/examples/with-ms-teams.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ grand_parent: Helm
 
 # ACS Helm Deployment with Microsoft Teams Connector
 
-The [Alfresco Microsoft Teams Connector](https://docs.alfresco.com/) enables
+The [Alfresco Microsoft Teams Connector](https://support.hyland.com/r/Alfresco/Alfresco-Process-Automation/7.17/Alfresco-Process-Automation/Model/Connectors/Microsoft-Teams-Connector) enables
```
Comment (gionn) [inline docs/helm/examples/with-ms-teams.md:9]: not the right one, can't find the equivalent of https://docs.alfresco.com/microsoft-teams/latest/

**Review by gionn — COMMENTED** [review]
* mention to update the links in case a new minor release is made
* consider if worth switching to the alternate link syntax to have all of them at the bottom of each markdown page

File: docs/helm/examples/with-ms-teams.md (hunk 1 above)
Comment (alxgomz) [reply docs/helm/examples/with-ms-teams.md:9]: The previous link pointed to the main doc page. Let's do the same here but to the new domain

**Review by alxgomz — APPROVED** [review]
urls didn't get shorter :)

**Review by alxgomz — APPROVED** [review]

**Review by gionn — APPROVED** [review]

**Pull request merged**